deployment/helm/crds/akri-configuration-crd.yaml

```diff
@@ -22,6 +22,12 @@ spec:
                       type: string
                     discoveryDetails:
                       type: string
+                    discoveryProperties:
+                      nullable: true
+                      type: array
+                      items: # {{DiscoveryProperty}}
+                        x-kubernetes-preserve-unknown-fields: true
```

Comment:
why not define the DiscoveryProperty schema here?

Reply:
yes, why not.  I'll update the crd